plugins_src/import_export/wpc_svg_path.erl

```diff
@@ -319,17 +318,17 @@ make_svg(Name, #svg_importer_params{
         wpa:pref_set(wpc_svg_path, svg_use_viewbox_coords, UseViewboxCoords),
         wpa:pref_set(wpc_svg_path, svg_viewbox_scale, SetViewBoxScale),
         {ok, E3dFile};
-    {error,Reason} ->
-        {error, ?__(1,"Inkscape .svg path import failed")++": " ++ Reason};
     E ->
         io:format("File Import Error Report:\n ~p\n",[E]),
         {error, ?__(2,"Inkscape .svg path import internal error")}
+    catch
+        error:Reason ->
+            {error, ?__(1,"Inkscape .svg path import failed")++": " ++ Reason}
```

Comment:
It's the **E** branch you will need to catch, the {error, Reason} is a normal return value.

Suggestion:   (ST = StackTrace so we can see where there error occured) 

```
catch Error:Reason:ST ->   
        io:format("File Import Error Report:\n ~p:~p\n  ~p\n",[Error, Reason, ST]),
        {error, ?__(2,"Inkscape .svg path import internal error")}
```